build.gradle.kts

```diff
@@ -2,26 +2,60 @@ plugins {
     java
     id("org.springframework.boot") version "3.2.5"
     id("io.spring.dependency-management") version "1.1.4"
+
+    val kotlinVersion = "1.9.22"
+    kotlin("jvm") version kotlinVersion
+    kotlin("kapt") version kotlinVersion
 }
 
 group = "com.wcc.cms"
 version = "0.0.1-SNAPSHOT"
 
+val mapStringVersion = "1.5.5.Final"
+
 java {
     sourceCompatibility = JavaVersion.VERSION_21
 }
 
+configurations {
+    compileOnly {
+        extendsFrom(configurations.annotationProcessor.get())
+    }
+}
+
 repositories {
     mavenCentral()
 }
 
 dependencies {
+
+    kapt("org.mapstruct:mapstruct-processor:$mapStringVersion")
+    implementation("org.mapstruct:mapstruct:$mapStringVersion")
+    annotationProcessor("org.projectlombok:lombok-mapstruct-binding:0.2.0")
+
+    compileOnly("org.projectlombok:lombok")
+    annotationProcessor("org.projectlombok:lombok")
+
+
     implementation("org.springframework.boot:spring-boot-starter-web")
-    implementation("io.springfox:springfox-boot-starter:3.0.0")
+    implementation("io.springfox:springfox-boot-starter:3.0.0") // WHAT WAS THAT AGAIN?
```

Comment:
I tried checking what this was -then explored, that this is a deprecated dependency to use the swagger then I started exploring something around this - https://springdoc.org/#properties
And Swagger document got generated automatically 

<img width="1661" alt="image" src="https://github.com/Women-Coding-Community/wcc-backend/assets/112942761/3964ff68-6a39-4604-8b93-ea694b2aa7ba">

Let me know if we should implement that ?
I just did small POC

Reply:
I think that would be nice. I don't know how this was merged, by me. Probably by mistake, I am going to revert. But please go ahead with this automatic generation, and please share with us how to do it.